tests/sentry/incidents/test_subscription_processor.py

```diff
@@ -285,6 +291,18 @@ def comparison_rule_below(self):
         self.trigger.update(alert_threshold=50)
         return rule
 
+    @cached_property
+    def dynamic_rule(self):
+        rule = self.rule
+        rule.update(
+            detection_type=AlertRuleDetectionType.DYNAMIC,
+            sensitivity=AlertRuleSensitivity.HIGH,
+            seasonality=AlertRuleSeasonality.AUTO,
+        )
+        # dynamic alert rules have a threshold of 0.0
```

Comment:
Do we check this in the serializer? If not, do you mind adding a ticket in the epic for us to make sure we do?

Reply:
[https://getsentry.atlassian.net/browse/ALRT-165](url)